boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -46,6 +52,36 @@
 	};
 };
 
+&fmc {
+	status = "okay";
+
+	pinctrl-0 = <
+		&fmc_sdcke1_pb5 &fmc_sdne1_pb6 &fmc_sdclk_pg8 &fmc_sdnwe_pc0
+		&fmc_sdnras_pf11 &fmc_sdncas_pg15 &fmc_a0_pf0 &fmc_a1_pf1
+		&fmc_a2_pf2 &fmc_a3_pf3 &fmc_a4_pf4 &fmc_a5_pf5 &fmc_a6_pf12
+		&fmc_a7_pf13 &fmc_a8_pf14 &fmc_a9_pf15 &fmc_a10_pg0 &fmc_a11_pg1
+		&fmc_a14_pg4 &fmc_a15_pg5 &fmc_nbl0_pe0 &fmc_nbl1_pe1 &fmc_d0_pd14
+		&fmc_d1_pd15 &fmc_d2_pd0 &fmc_d3_pd1 &fmc_d4_pe7 &fmc_d5_pe8
+		&fmc_d6_pe9 &fmc_d7_pe10 &fmc_d8_pe11 &fmc_d9_pe12 &fmc_d10_pe13
+		&fmc_d11_pe14 &fmc_d12_pe15 &fmc_d13_pd8 &fmc_d14_pd9
+		&fmc_d15_pd10>;
+};
+
+&fmc_sdram {
```

Comment:
this should be embedded in `fmc` node (no need to use the label), see the FMC PR.

Reply:
Yeah I wen't over your pull request. Makes sense.